frontend/app/src/composables/assets/retrieval.ts

```diff
@@ -154,13 +154,15 @@ export const useAssetInfoRetrieval = () => {
       );
       return result;
     } catch (e: any) {
-      notify({
-        title: t('actions.assets.erc20.error.title', payload),
-        message: t('actions.assets.erc20.error.description', {
-          message: e.message
-        }),
-        display: true
-      });
+      if (!(e instanceof UserCancelledTaskError)) {
```

Comment:
After I think again, is it makes sense to move this to a function instead?
Something like `processError` that returns boolean

Reply:
and also if possible, use return early.
`if(!processError(e)) return`

Reply:
done